src/main/java/software/amazon/awssdk/crt/mqtt5/packets/PublishPacket.java

```diff
@@ -449,6 +449,30 @@ public PublishPacketBuilder withUserProperties(List<UserProperty> userProperties
          */
         public PublishPacketBuilder() {}
 
+        /**
+         * Creates a new PublishPacketBuilder with most common parameters set.
```

Comment:
Trivial: Both are labeled "most common" parameter set. Maybe we remove "most" from both.

Separately, do we wan't both? I thought we were only doing one new constructor with topic, and QoS. The parameter in question related to necessity I believe was payload. e.g. Some SDKs allow a publish to be created without a payload but QoS is always necessary.